State-of-Rhode-Island-eCMS/ecms_profile
# Riga 576/press release syndication

## Summary / Approach
Adds an install hook to automatically add the hub site as an endpoint for press release syndication.

## Testing Instruction(s)
Install the `ecms_api_press_release_publisher` module and confirm that the hub site is created as a syndicate.

## Screenshots
n/a

## Metadata
| Question | Answer |
|----------|--------|
| Did you apply meaningful labels to the pull request? |yes
| Documentation reflects changes? |no
| `CHANGELOG` reflects changes? |yes
| Unit/Functional tests cover changes? |no
| Did you perform browser testing? |yes
| Did you provide detail in the summary on how and where to test this branch? |yes
| Risk level |low
| Relevant links | [RIGA-576](https://thinkoomph.jira.com/browse/RIGA-576)
